src/main/java/roomescape/dao/ReservationDAO.java

```diff
@@ -0,0 +1,61 @@
+package roomescape.dao;
+
+import java.util.List;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import roomescape.domain.Reservation;
+
+import java.sql.*;
+
+@Repository
+public class ReservationDAO {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    public ReservationDAO(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public void create(Reservation reservation) {
+        String sql = "INSERT INTO reservation (name, date, time) VALUES (?, ?, ?)";
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+
+        int update = jdbcTemplate.update(connection -> {
+            PreparedStatement ps = connection.prepareStatement(sql, Statement.RETURN_GENERATED_KEYS);
+            ps.setString(1, reservation.getName());
+            ps.setString(2, reservation.getDate());
+            ps.setString(3, reservation.getTime());
+
+            return ps;
+        }, keyHolder);
+
+        Number key = keyHolder.getKey();
+        if (key != null) {
+            reservation.setId(key.longValue());
+        }
+    }
+
+    public List<Reservation> findAll() {
+        String sql = "SELECT * FROM reservation";
+
+        List<Reservation> reservations = jdbcTemplate.query(
+                sql, (resultSet, rowNum) -> new Reservation(
+                        resultSet.getLong("id"),
+                        resultSet.getString("name"),
+                        resultSet.getString("date"),
+                        resultSet.getString("time")
+                ));
+        return reservations;
+    }
+
+    public void deleteById(Long id) throws SQLException {
+        String sql = "delete from reservation where id=?";
+        int update = jdbcTemplate.update(sql, id);
+        System.out.println("update = " + update);
+    }
+
+
+
+}
```

Comment:
```suggestion
    }
}
```

이런 부분도 깔끔하게 챙겨주시면 좋습니답